### PR TITLE
Add `dms-viz` configuration to the pipeline

### PR DESCRIPTION
**Pull request by WillHannon-MCB**
I've added rules to the pipeline to make the configuration file for visualizing the mutation fitness data with dms-viz. I'm tracking the configuration file in a `dms-viz` subdirectory of the results directories. 

I'll add a link to `dms-viz` with the data pre-loaded after this branch is merged because the path to the data depends on the branch.

### CODE REVIEW

**Review by jbloom — CHANGES_REQUESTED** [review]
@WillHannon-MCB, mostly looks good. Just two things:

1. Can you add a README to the `./data/structures/` subdirectory that describes the process used to obtain or generated each PDB so we know where those came from?
2. Not sure if this is going to be fixed later, but the link the main README to the `dms-viz` visualizations is currently empty.

**Comment by WillHannon-MCB** [comment]
I added documentation describing each structure. 

The link is left empty intentionally. The path to the data depends on the branch, so I can't generate the link to dms-viz until the branches have been merged.

**Review by jbloom — APPROVED** [review]
@WillHannon-MCB, I'm going ahead and approving now.

**Pull request merged**
